Enyim.Caching/Enyim.Caching.csproj

```diff
@@ -2,14 +2,19 @@
 
   <PropertyGroup>
     <Description>CWMemcache is a Memcached client library for .NET Core. Usage: Add services.AddEnyimMemcached(...) and app.UseEnyimMemcached() in Startup. Add IMemcachedClient into constructor.</Description>
-    <VersionPrefix>2.0.9</VersionPrefix>
+    <VersionPrefix>2.1.0</VersionPrefix>
     <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <AssemblyName>CWMemcache</AssemblyName>
     <PackageId>CWMemcache</PackageId>
     <PackageTags>memcached;cache</PackageTags>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>latest</LangVersion>
+    <PackageReleaseNotes>
+      2.1.0
+      Add ActivitySource
```

Comment:
Add what is the purpose of adding this

Reply:
Write proper release notes. Mention why this is getting added.